<ui version="4.0" >
 <class>PoDoFoBrowserBase</class>
 <widget class="Q3MainWindow" name="PoDoFoBrowserBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PoDoFo Browser</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>42</y>
     <width>758</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QTreeView" name="listObjects" >
      <property name="mouseTracking" >
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter3" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layout3" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Contents of the associated stream:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStream" >
           <property name="text" >
            <string/>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Q3TextEdit" name="textStream" >
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="buttonImport" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>&amp;Import Stream ...</string>
             </property>
             <property name="shortcut" >
              <string>Alt+I</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonExport" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>&amp;Export Stream ...</string>
             </property>
             <property name="shortcut" >
              <string>Alt+E</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>241</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Q3ToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="label" >
    <string>Tools</string>
   </property>
   <addaction name="fileNewAction" />
   <addaction name="fileOpenAction" />
   <addaction name="fileSaveAction" />
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="editCutAction" />
    <addaction name="editnew_itemAction" />
    <addaction name="editPasteAction" />
    <addaction name="separator" />
    <addaction name="editInsertObjectAction" />
    <addaction name="editInsertKeyAction" />
    <addaction name="separator" />
    <addaction name="editDeleteObjectAction" />
    <addaction name="editDeleteKeyAction" />
    <addaction name="separator" />
    <addaction name="editLoadAllObjectsAction" />
   </widget>
   <widget class="QMenu" name="Tools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="toolsToHexAction" />
    <addaction name="toolsFromHexAction" />
   </widget>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewAction" />
    <addaction name="fileOpenAction" />
    <addaction name="fileSaveAction" />
    <addaction name="fileSaveAsAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="Edit" />
   <addaction name="Tools" />
   <addaction name="helpMenu" />
  </widget>
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset>image0</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="iconText" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset>image1</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="iconText" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAction" >
   <property name="icon" >
    <iconset>image2</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="iconText" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save As</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="filePrintAction" >
   <property name="icon" >
    <iconset>image3</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="toolsToHexAction" >
   <property name="text" >
    <string>Convert &amp;to Hexstring</string>
   </property>
   <property name="iconText" >
    <string>Convert &amp;to Hexstring</string>
   </property>
  </action>
  <action name="toolsFromHexAction" >
   <property name="text" >
    <string>Convert &amp;from Hexstring</string>
   </property>
   <property name="iconText" >
    <string>Convert &amp;from Hexstring</string>
   </property>
  </action>
  <action name="Action" >
   <property name="iconText" >
    <string>Action</string>
   </property>
  </action>
  <action name="editCutAction" >
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="iconText" >
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="editnew_itemAction" >
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="iconText" >
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="editPasteAction" >
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="iconText" >
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="editInsertObjectAction" >
   <property name="text" >
    <string>Insert &amp;Object</string>
   </property>
   <property name="iconText" >
    <string>Insert &amp;Object</string>
   </property>
  </action>
  <action name="editInsertKeyAction" >
   <property name="text" >
    <string>Insert &amp;Key</string>
   </property>
   <property name="iconText" >
    <string>Insert &amp;Key</string>
   </property>
  </action>
  <action name="editDeleteObjectAction" >
   <property name="text" >
    <string>&amp;Delete Object</string>
   </property>
   <property name="iconText" >
    <string>&amp;Delete Object</string>
   </property>
  </action>
  <action name="editDeleteKeyAction" >
   <property name="text" >
    <string>D&amp;elete Key</string>
   </property>
   <property name="iconText" >
    <string>D&amp;elete Key</string>
   </property>
  </action>
  <action name="editLoadAllObjectsAction" >
   <property name="text" >
    <string>&amp;Load all objects</string>
   </property>
   <property name="iconText" >
    <string>&amp;Load all objects</string>
   </property>
   <property name="toolTip" >
    <string>Objects are loaded on demand from the PDF file. This action will load all objects immediately which can take some time.</string>
   </property>
  </action>
  <action name="actionScan_Tree" >
   <property name="text" >
    <string>Scan Tree</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3ToolBar</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fileNewAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>fileNew()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>fileSave()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAsAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>fileSaveAs()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePrintAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>filePrint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExitAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>fileExit()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAboutAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolsToHexAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>toolsToHex()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolsFromHexAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>toolsFromHex()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDeleteKeyAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>editDeleteKey()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDeleteObjectAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>editDeleteObject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editInsertKeyAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>editInsertKey()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editInsertObjectAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>editInsertObject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLoadAllObjectsAction</sender>
   <signal>activated()</signal>
   <receiver>PoDoFoBrowserBase</receiver>
   <slot>loadAllObjects()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
